use_cases/eluc/predictors/demo_predictors.ipynb

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
This file shows off the workflow. It instantiates a model, fits, predicts, and saves it. Then it instantiates another one, loads, predicts, and compares the 2 outputs to show they are the same.